src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.ts

```diff
@@ -78,7 +93,17 @@ export class DotEditPageNavComponent implements OnChanges {
                 label: this.dotMessageService.get('editpage.toolbar.nav.content'),
                 link: 'content'
             },
-            this.getTemplateNavItem(dotRenderedPage, enterpriselicense)
+            this.getTemplateNavItem(dotRenderedPage, enterpriselicense),
+            {
+                needsEntepriseLicense: false,
+                disabled: false,
+                icon: 'fa fa-plus',
+                label: this.dotMessageService.get('editpage.toolbar.nav.properties'),
+                action: (event, inode, label) => {
```

Comment:
Do this here

```
+            this.dotContentletEditorService.edit({
+                data: {
+                    inode: buttonClicked.inode
+                }
+            });
```

and don't emit anything, just do the action and that's it.

In the menu on click call the action function if exist. That's how primeng does it and looks cleaner.